lib/recording/recordings.ex

```diff
@@ -0,0 +1,168 @@
+defmodule Onvif.Recording.Recordings do
```

Comment:
Is this a duplicate of `Onvif.Recording.Recording`?

Reply:
yes, I probably forgot to add the file when I was renaming. Cleaning up from all PRs now